net-libs/xdp-tools/files/1.3.1-no-Werror.patch

```diff
@@ -0,0 +1,97 @@
+
+Bug: https://bugs.gentoo.org/899744
+
+diff --git a/configure b/configure
+index 62c2f88..230637f 100755
+--- a/configure
++++ b/configure
+@@ -249,7 +249,7 @@ int main(int argc, char **argv) {
+     return 0;
+ }
+ EOF
+-    compile_cmd="$CC -o $TMPDIR/libbpftest $TMPDIR/libbpftest.c -Werror $LIBBPF_CFLAGS $LIBBPF_LDLIBS"
++    compile_cmd="$CC -o $TMPDIR/libbpftest $TMPDIR/libbpftest.c $LIBBPF_CFLAGS $LIBBPF_LDLIBS"
```

Comment:
Yeah, we probably should've kept that one, or refined the -Werror instead. Thanks for the heads up.

Reply:
:sob:

Reply:
It's my fault for not noticing!

Reply:
@thesamesam I'll make a fixup PR..

Reply:
https://github.com/gentoo/gentoo/pull/30586 -- hopefully this won't bring us back to where we started.

Reply:
What's the reason you need to remove -Werror in the first place? Anything we can fix in upstream? Also, why are you patching the Makefiles instead of just setting CFLAGS in the environment when building?

Reply:
@tohojo In a nutshell: https://bugs.gentoo.org/show_bug.cgi?id=Werror - because -Werror frequently breaks things for no good reason, and Gentoo is always the first to find out, e.g. on gcc or other toolchain updates like building the world with clang etc.
I actually wanted to start a question/discussion and try to convince you to remove -Werror if possible, but just didn't get to it yet because $WORK.